router/auth/provider.go

```diff
@@ -194,6 +197,9 @@ func defaultCallbackHandler(p Provider, oac *oauth2.Config, repo repository.Repo
 					Extra:        model.JSON{"externalName": tu.GetRawName()},
 				},
 			}
+			if err := vd.Validate(args.Name, validator.UserNameRuleRequired...); err != nil {
+				return herror.BadRequest("Your Slack display name doesn't match with traQ ID format")
```

Comment:
他のものと共通して使っているので一応こうしておいた方が良いかなって思います
ユーザーにとって少し分かりづらいかもしれないですが... （todo #826 ）
```suggestion
				return herror.BadRequest("Your name doesn't match with traQ ID format")
```